examples/nest/ArtificialSynchrony.sli

```diff
@@ -163,7 +161,7 @@ userdict begin
       /overwrite_files true          % overwrite previous output
   >> SetKernelStatus
 
-  sim 1 eq { /neurons /iaf_psc_alpha_canon nr Create def }
+  sim 1 eq { /neurons /iaf_psc_alpha_ps nr Create def }
```

Comment:
This example is not running for me, you probably need to change a `tau_syn` somewhere!

Reply:
Done in fc1357d